### PR TITLE
add the no EORG popup

### PR DESCRIPTION
**Pull request by MilonPL**
## About the PR

idea shamelessly stolen from harmony except actually good (xaml gaming)

## Why / Balance

we ban people for EORG now, it's only fair if players get a reminder

## Media

image

## Breaking changes

**Changelog**

:cl:
- add: Added a no EORG reminder to the end of round!

### CODE REVIEW

File: Content.Client/DeltaV/RoundEnd/NoEorgPopup.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
+using Content.Client.UserInterface.Controls;
+using Content.Shared.DeltaV.CCVars;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Configuration;
+using Robust.Shared.Timing;
+using Robust.Shared.Utility;
+
+namespace Content.Client.DeltaV.RoundEnd
```
Comment (deltanedas) [inline Content.Client/DeltaV/RoundEnd/NoEorgPopup.xaml.cs]: file scoped namespace noob

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Content.Client/DeltaV/RoundEnd/NoEorgPopup.xaml.cs (hunk 1 above)
Comment (MilonPL) [reply Content.Client/DeltaV/RoundEnd/NoEorgPopup.xaml.cs]: ok wtf how did that happen

**Review by deltanedas — APPROVED** [review]

**Comment by deltanedas** [comment]
liltenhead ops

**Comment by BlitzTheSquishy** [comment]
Speed merge it RIGHT NOW

**Comment by MilonPL** [comment]
unrelated test fail ops twice
i should get the bypass checks permission trust

**Pull request merged**